applications/HDF5Application/python_scripts/initialization_from_hdf5_process.py

```diff
@@ -91,7 +90,8 @@ def CreateCoreSettings(user_settings):
     for key in user_settings["file_settings"]:
         core_settings[0]["io_settings"][key] = user_settings["file_settings"][key]
     core_settings[0]["file_access_mode"] = "read_only"
-    if IsDistributed():
+    model_part_name = user_settings["model_part_name"]
+    if model[model_part_name].IsDistributed():
```

Comment:
I am not agaisnt this. Just curious, is this the proper way of checking distributed runs? Are we not gonna use global "IsDistributed" anymore? @philbucher

Reply:
nope
imagine you are running in MPI but one of your solvers doesn't use MPI
Example: FSI where the structure runs in OpenMP

It is quite specific but it will probably be used more in the future
This also means that `MPI_COMM_WORLD` should not be used directly! => the HDFApp needs to be refactored for this